app/pages/sections/routes.js

```diff
@@ -20,6 +20,7 @@ import { additionalServicesRoutes } from './order-items/additional-services/rout
 import { associatedServicesRoutes } from './order-items/associated-services/routes';
 import { getFundingSource } from '../../helpers/api/ordapi/getFundingSource';
 import { putFundingSource } from '../../helpers/api/ordapi/putFundingSource';
+import { getOrderDescription } from '../../helpers/api/ordapi/getOrderDescription';
```

Comment:
I think we probably want to use the `getOrderDescription` that I've added as part of activity 8866 here, so this should be:

```javascript
import { getOrderDescription } from '../../helpers/routes/getOrderDescription';
```